doc/SpectatorView.Setup.md

```diff
@@ -147,9 +155,9 @@ If you are building Azure Spatial Anchors on iOS, you will need to take some add
 
 5. Make sure to declare the SpectatorView.Android scene as the scene included. If SpectatorView.Android does not exist in your list of scenes to choose from in the build settings, open the scene in the Unity editor. Then reopen the build settings and press 'Add Open Scenes'.
 
-![Marker](images/AndroidSceneSelection.png)
+![Marker](images/AndroidSpectatorViewExampleBuildSettings.png)
 
-6. Build and deploy the application. If you're targeting a new Android development device, you may need to enable developer options and debugging. For information on setting up your Android device for debugging, see [here](https://developer.android.com/studio/debug/dev-options).
+6. `Build and Run` the application through Unity's Build Settings. If you're targeting a new Android development device, you may need to enable developer options and debugging. For information on setting up your Android device for debugging, see [here](https://developer.android.com/studio/debug/dev-options).
 

```

Comment:
Might be good to also mention that you may need to hit the "refresh" button to get the device to show up in Unity.  I'm assuming the build view screenshot has the right view, and can refer to that...I just can't see the screenshot in the PR review.

Reply:
i added another screen shot here with the device selector.